elasticgraph-indexer/lib/elastic_graph/indexer/datastore_indexing_router.rb

```diff
@@ -239,7 +241,10 @@ def source_event_versions_in_index(operations)
           errors = msearch_response.fetch("responses").select { |res| res["error"] }
 
           if errors.empty?
-            versions_by_op = ops.zip(msearch_response.fetch("responses")).to_h do |(op, response)|
+            # We assume the size of the ops and the other array is the same and it cannot have `nil`.
+            zip = ops.zip(msearch_response.fetch("responses")) #: Array[[_Operation, Hash[String, Hash[String, untyped]]]]
```

Comment:
```suggestion
            zip = ops.zip(msearch_response.fetch("responses")) # : ::Array[[_Operation, ::Hash[::String, ::Hash[::String, untyped]]]]
```

The lack of a space after `#` is failing the build because of our linter.